locksmith/package.json

```diff
@@ -41,6 +41,7 @@
     "@types/express": "4.17.0",
     "@types/express-serve-static-core": "4.16.7",
     "@types/jest": "24.0.14",
+    "@types/nock": "^10.0.3",
```

Comment:
I think we want to avoid the caret versions and just have pinned deps (and do dep installs separate from usual code)